plutus-contract/src/Language/Plutus/Contract/Wallet.hs

```diff
@@ -181,6 +181,6 @@ addOutputs pk vl tx = tx & over Tx.outputs (pko :) where
 
 -- | Balance an unabalanced transaction, sign it, and submit
 --   it to the chain in the context of a wallet.
-handleTx :: (Members WalletEffects effs, Member Log effs, Member (Error WalletAPIError) effs) => UnbalancedTx -> Eff effs Tx
+handleTx :: (Member WalletEffect effs, Member ChainIndexEffect effs, Member SigningProcessEffect effs, Member Log effs, Member (Error WalletAPIError) effs) => UnbalancedTx -> Eff effs Tx
```

Comment:
`WalletEffects` was actually too restrictive - it includes a `NodeClientEffect` which we don't need for this.